build.gradle

```diff
@@ -77,6 +77,10 @@ dependencies {
     compileOnly "com.github.spotbugs:spotbugs-annotations:${spotbugs.toolVersion.get()}"
     implementation "org.slf4j:slf4j-api:1.7.+"
     implementation "org.slf4j:slf4j-log4j12:1.7.+"
```

Comment:
We should create a separate task in order to upgrade this to the latest 2.x version of the log4j library in order to remove a bunch of critical, high and medium CVE vulerabilities

Reply:
https://github.com/OwnZones/photon/pull/12